packages/api/src/substrate/index.ts

```diff
@@ -370,21 +371,23 @@ export class SubsocialSubstrateApi {
   /**
    * Find and load next space id from Subsocial blockchain.
    *    
-   * @returns An integer of the next space id.
+   * @returns An string of the next space id.
    * 
    */  
-  async nextSpaceId (): Promise<SpaceId> {
-    return this.querySpaces('nextSpaceId')
+  async nextSpaceId (): Promise<string> {
+    const id = await this.querySpaces('nextSpaceId')
+    return asString(id)
   }
 
   /**
    * Find and load next post id from Subsocial blockchain.
    *    
-   * @returns An integer of the next post id.
+   * @returns An string of the next post id.
```

Comment:
The same situation as above.

Reply:
Updated [b227542](https://github.com/dappforce/subsocial-js/pull/129/commits/b22754252ad8e73d2eda511f5f8eb49a907c643b)